docs/Installation.md

```diff
@@ -101,6 +104,12 @@ Note: In case software-discovery-tool code is already checked out, do the follow
 ```cd software-discovery-tool/src/tests``` <br />
 ```pytest```
 
+### Step 6: Set read/write permissions on /opt/software-discovery-tool/ folder if you are getting PytestCacheWarning when running pytest
+
+        sudo chown -R user:user /opt/software-discovery-tool/
```

Comment:
This command actually recursively changes the ownership of your files, but according to the instructions earlier we want the apache user to own these files (see line 92).

That said, since running pytest is an alternative option to navigating to the URL in the browser, I wouldn't make this another step. Instead, I'd recommend using the following to run pytest as the apache user:

```sudo -u apache pytest```

So I'd just suggest that you use that in the instructions instead.

Reply:
Yes this seems better